tf/modules/oonith_service/main.tf

```diff
@@ -0,0 +1,222 @@
+locals {
+  name = "oonith-service-${var.service_name}"
+}
+
+resource "aws_iam_role" "oonith_service_task" {
+  name = "${local.name}-task-role"
+
+  tags = var.tags
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ecs-tasks.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "oonith_service_task" {
+  name = "${local.name}-task-role"
+  role = aws_iam_role.oonith_service_task.name
+
+  policy = templatefile("${path.module}/templates/profile_policy.json", {})
+}
+
+resource "aws_cloudwatch_log_group" "oonith_service" {
+  name = "ooni-ecs-group/${local.name}"
+}
+
+
+locals {
+  container_port = 80
```

Comment:
This port is actually at the moment configured to be port 8080: https://github.com/ooni/backend/pull/827/files#diff-15056f69005bdac43028da51801647dc54a1831e445bdb79def725f9e1a65c38R21.

This is actually a bit annoying and I'm planning to change the definition of the oohelperd service to bind to port 80 only and not have this port based separation between oohelperd service and prometheus metrics endpoint.

The prometheus should also bind to the same port and implement HTTP Basic auth to expose the metrics endpoint similar to how we do it in ooniapi services, see: https://github.com/ooni/backend/blob/master/ooniapi/common/src/common/metrics.py#L18

Reply:
This is done